docs/feature-gates.md

```diff
@@ -32,25 +32,26 @@ edit the Agent configuration in the
-| `SupportBundleCollection` | Agent + Controller | `false` | Alpha | v1.10         | N/A          | N/A        | Yes                |       |
+| Feature Name              | Component          | Default | Stage | Alpha Release | Beta Release | GA Release | Extra Requirements | Notes                                          |
+| ------------------------- | ------------------ | ------- | ----- | ------------- | ------------ | ---------- | ------------------ | ---------------------------------------------- |
+| `AntreaProxy`             | Agent              | `true`  | Beta  | v0.8          | v0.11        | N/A        | Yes                | Must be enabled for Windows.                   |
+| `EndpointSlice`           | Agent              | `false` | Alpha | v0.13.0       | N/A          | N/A        | Yes                |                                                |
+| `TopologyAwareHints`      | Agent              | `false` | Alpha | v1.8          | N/A          | N/A        | Yes                |                                                |
+| `AntreaPolicy`            | Agent + Controller | `true`  | Beta  | v0.8          | v1.0         | N/A        | No                 | Agent side config required from v0.9.0+.       |
+| `Traceflow`               | Agent + Controller | `true`  | Beta  | v0.8          | v0.11        | N/A        | Yes                |                                                |
+| `FlowExporter`            | Agent              | `false` | Alpha | v0.9          | N/A          | N/A        | Yes                |                                                |
+| `NetworkPolicyStats`      | Agent + Controller | `true`  | Beta  | v0.10         | v1.2         | N/A        | No                 |                                                |
+| `NodePortLocal`           | Agent              | `true`  | Beta  | v0.13         | v1.4         | N/A        | Yes                | Important user-facing change in v1.2.0         |
+| `Egress`                  | Agent + Controller | `true`  | Beta  | v1.0          | v1.6         | N/A        | Yes                |                                                |
+| `NodeIPAM`                | Controller         | `false` | Alpha | v1.4          | N/A          | N/A        | Yes                |                                                |
+| `AntreaIPAM`              | Agent + Controller | `false` | Alpha | v1.4          | N/A          | N/A        | Yes                |                                                |
+| `Multicast`               | Agent              | `false` | Alpha | v1.5          | N/A          | N/A        | Yes                |                                                |
+| `Multicluster`            | Agent + Controller | `false` | Alpha | v1.7          | N/A          | N/A        | Yes                | Controller side config required from v1.10.0+. |
```

Comment:
Probably say: Controller side feature gate added in v1.10.0.

Reply:
Done